### DISCUSSION TITLE
User consent vs. PreLogon

### DISCUSSION THREAD

**Discussion by esgheorghe** [Ylianst/MeshCentral#6426] [Q&A]
Hello,

is there a way to configure the user consent so that the user has to agree when connecting, but if no one is logged in (prelogon) the connection is made without user consent.

I use the latest MeshCentral version (1.1.32), the agent is installed and starts automatically.

Many thanks & best regards
Stefan

**Comment by si458** [comment]
this has already been asked a lot
NOT CURRENTLY NO
the is already an open issue asking for this feature tho 
will tag here when i find it

**Reply by esgheorghe** [reply]
thanks for fast anwser.

Hopefully, the feature will be available soon

How can i help?

**Reply by tobias9931** [reply]
#6158
